src/index.html

```diff
@@ -243,9 +246,6 @@ <h5 class="modal-title" id="exampleModalLabel">Get Vowel Sounds, Encrypted</h5>
 </div>
 
 <aside id="declaration">
-    >>> ^C
```

Comment:
wha... why?

Reply:
I thought that the `Keyboard Interrupts` are a mistake.
Sorry, I will put that back in.